src/store/PriceLineStore.ts

```diff
@@ -175,8 +178,11 @@ export default class PriceLineStore {
     }
 
     get priceLineWidth() {
-        // TODO: measure the price label width
-        return Math.max(this.priceDisplay.length * 8, 60);
+        return window.flutterChart?.app.getCurrentTickWidth() || 60;
+    }
+
+    get overlappedBarrierWidth(): number {
+        return 16;
```

Comment:
if this getter always returns 16, maybe it'll make sense to create a constant?

Reply:
Yeah, but I don't want to change this as there might be a need to make this dynamic again in the future and someone will have to figure it out and write it up again from scratch, so let's leave it here.